site/pages/docs/ref/provisional-fr.hbs

```diff
@@ -113,3 +119,27 @@
 	&lt;/div&gt;
 &lt;/div&gt;</code></pre>
 </div>
+
+<h3 id="well-header-rwd-bgimg">Showing random image using [data-wb-randomize] plugin</h3>
+<p>Below example shows one image out of three images randomly. Refresh the page to see that the image changes.</p>
+<dl>
+	<dt><code>shuffle</code> is a boolean value and if it is true, it will shuffle the order of child components (in this example, the child components are three <code>img</code>s).</dt>
+	<dt><code>selector</code> specifies the type of child components to be displayed.</dt>
+	<dt><code>number</code> specifies the number of child components to be displayed. If it is more than the number of child components, all of them will be displayed.</dt>
+	<dt><code>toggle</code> is a boolean value and if it is true, it will toggle 'hide' class of child components to reveal (in this example, the child components are three <code>img</code>s).</dt>
+	<dt><code>toggleClass</code> specifies the class that will be toggled.</dt>
```

Comment:
Merge toggle and toggleClass togetter,

So the toggle setting will contains the CSS class to toggle,